Source/Dafny/ConcreteSyntax/ConcreteSyntaxTree.cs

```diff
@@ -122,7 +128,8 @@ public T Append<T>(T node)
       BraceStyle open = BraceStyle.Space,
       BraceStyle close = BraceStyle.Newline) {
       Contract.Requires(header != null);
-      Append(ConcreteSyntaxTreeUtils.Block(out ConcreteSyntaxTree result, header, footer, open, close));
+      Append(ConcreteSyntaxTreeUtils.Block(out ConcreteSyntaxTree result, header: header, footer: footer, open: open,
```

Comment:
Are the changes on this line useful?

Reply:
This might just be IDE auto-formatting?

Reply:
Yes, it was, I thought as it's a function call it will be fine. should I let it be as it is or revert it back as before this change ?